src/components/ha-sidebar.ts

```diff
@@ -303,6 +303,9 @@ class HaSidebar extends LitElement {
     if (changedProps.has("alwaysExpand")) {
       this.expanded = this.alwaysExpand;
     }
+
+    this._rtl = computeRTL(this.hass);
```

Comment:
We should only calculate this if `changedProps.has("hass")`. And please merge that with below if. So first check if hass changed, then return if it has not. Then recompute RTL, then check if scroll is supported…